index.cjs

```diff
@@ -119,10 +121,32 @@ function copyQuickstarts() {
     path.join(replicacheQuickstartsDest, 'package.json'),
     JSON.stringify(packageJson, null, 2) + os.EOL
   )
+
+}
+
+function createLicense() {
+  const rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout,
+  });
+
+  rl.question(
+    'Do you want to create a Replicache license? (y/n) ',
```

Comment:
Maybe better to ask if they already have one? They might not realize that to use replicache they have to create one whether they like it or not.

Reply:
Good idea, what do you think about 
`Do you need to get a Replicache license? (y/N)`